### PR TITLE
$latestVersion function

### PR DESCRIPTION
**Pull request by LightnerDev**
## Type
- [ ] Bug Fix
- [x] Functions: `$latestVersion` (new)
- [ ] Callbacks: \<Callback Name>
- [ ] Handlers: \<Handler Type>
- [ ] Others: \______

Dependencies (Third Party Modules) needed: [axios](https://npmjs.com/package/axios) Size: 0.371MB (Very much lesser than 20MB)

Want a credit? Discord tag or other social media link: Lightner#5294

Referenced Issue: #NaN (Answer for an issue, if any)

## Description
Added new Function `$latestVersion`
Moved 1 ([axios](https://npmjs.com/package/axios)) dependency from "devDependency" to "dependency"

### Reason
This would be very helpful and make commands such as this: 
```js
bot.command({
  name: "check-for-new-version",
  code: `
  $if[$packageVersion==$latestVersion]
  No new updates found!
  $else
  New update found
  Installing!
  $exec[npm i aio.js@latest]
  `
})
```
possible.

### CODE REVIEW

**Comment by soudblox** [comment]
axios is already in devDependencies of aoi.js (just to tell you)

**Comment by LightnerDev** [comment]
> axios is already in devDependencies of aoi.js (just to tell you)

i said
> Moved 1 (axios) dependency from "devDependency" to "dependency"

so i moved it not added it meaning i knew it was in devDependency

**Comment by soudblox** [comment]
yeah, didn't see

**Comment by soudblox** [comment]
this function can broke if the api is down though

**Comment by LightnerDev** [comment]
true
maybe you guys can update this command each time there is a new update?
lemme edit the file to show you

**Comment by soudblox** [comment]
no, the developers would have to change the code every new update, and that... can be annoying even though it's a small change...

**Comment by soudblox** [comment]
the api is rarely down, so yeah, rollback to the previous commit instead

i was just saying that if the api is down, it can broke, as well as other codes that uses the api, i didn't say to change it

**Comment by LightnerDev** [comment]
sorry i was busy

File: package/functions/funcs/latestVersion.js [hunk 1]
```diff
@@ -1,15 +1,12 @@
+const axios = require('axios')
+
 module.exports = async d => {
-  /*axios.get('https://api.leref.ga/package/version')
+  axios.get('https://api.leref.ga/package/version')
```
Comment (Kino7916) [inline package/functions/funcs/latestVersion.js:4]: It's better to use the [npm Registry](https://registry.npmjs.org) instead of Third party API.

**Comment by soudblox** [comment]
```js
axios.get('https://registry.npmjs.org/aoi.js').then(a => a.data["dist-tags"].latest)
```

use that instead

**Comment by Kino7916** [comment]
We already had our own `request` api in handlers folder, you can use that instead of axios.

**Comment by Mr-Catty** [comment]
This branch has conflicts that must be resolved
Only those with write access to this repository can merge pull requests.

**Comment by LightnerDev** [comment]
> We already had our own `request` api in handlers folder, you can use that instead of axios.

can you teach me how to use it?
i cant figure it out

and to other ppl dont merge yet 😄

**Pull request closed without merging**